docs/BR.md

```diff
@@ -1008,6 +1043,62 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
 
+#### 3.2.2.9 Multi-Perspective Issuance Corroboration
+
+Multi-Perspective Issuance Corroboration attempts to corroborate the determinations made by the Primary Network Perspective from multiple additional Network Perspectives before Certificate issuance. This process can improve protection against equally-specific prefix Border Gateway Protocol (BGP) attacks or hijacks.
+
+The response returned from each corroborating Network Perspective MUST allow the CA to verify:
+- a. the presence of the expected 1) Random Value, 2) Request Token, 3) IP Address, or 4) Contact Address, as required by the relied upon validation method specified in Sections 3.2.2.4 and 3.2.2.5; and
+- b. the CA's authority to issue to the requested domain(s), as specified in Section 3.2.2.8.
+
+[Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address) describe the validation methods that require the use of Multi-Perspective Issuance Corroboration and how a Network Perspective can corroborate the outcomes determined by the Primary Network Perspective. A CA MAY use corroborating evidence for CAA record compliance for a period equal to the period of validity for Validation of Domain Authorization or Control specified in [Section 4.2.1](#421-performing-identification-and-authentication-functions).
+
+Results or information obtained from one Network Perspective MUST NOT be reused or cached when performing validation through subsequent Network Perspectives (e.g., different Network Perspectives cannot rely on a shared DNS cache to prevent an adversary with control of traffic from one Network Perspective from poisoning the DNS cache used by other Network Perspectives). The network infrastructure providing Internet connectivity to a Network Perspective MAY be administered by the same organization providing the computational services required to operate the Network Perspective.
+
+CAs MAY immediately retry Multi-Perspective Issuance Corroboration using the same validation method or an alternative method (e.g., a CA can immediately retry validation using "Email to DNS TXT Contact" if "Agreed-Upon Change to Website - ACME" does not corroborate the outcome of Multi-Perspective Issuance Corroboration). When retrying Multi-Perspective Issuance Corroboration, CAs MUST NOT rely on corroborations from previous attempts. There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
+
+The "Quorum Requirements" Table describes quorum requirements related to Multi-Perspective Issuance Corroboration. Network Perspectives are considered distinct when the straight-line distance between the two States, Provinces, or Countries they reside in is at least 500 km. Network Perspectives are considered "remote" when they are distinct from the Primary Network Perspective.
+
+Table: Quorum Requirements
```

Comment:
**Validation Subcommittee Comment (summarized):** The required number of remote perspectives needs to be clarified. 

**Response:** 

The "Quorum Requirements" table describes the maximum number of "non-corroborations" given the number of distinct remote network perspectives used. The end of 3.2.2.9 presents the "Phased Implementation Timeline," which describes implementation milestones that strengthen over time. Over the proposed implementation timeline, quorum requirements increase from undefined (during the period where CAs SHOULD be implementing MPIC but are not otherwise required) to 2+ (December 2024), to 3+ (December 2025), to 5+ (December 2026).

I've tried summarizing the phasing below, with changes across phases highlighted in `code block` formatting.

------


#### Phase 1: June 15, 2024 - December 14, 2024 ("CAs SHOULD implement MPIC"):
- **MUST implement MPIC?** No ("SHOULD" not "MUST")
- **Required quorum?:** N/A (MPIC not required)
- **MUST block issuance if the quorum is not met:**  N/A (MPIC not required)
- **Additional geographic diversity requirements?:** N/A (MPIC not required)

------

#### Phase 2: December 15, 2024 - June 14, 2025 ("CAs MUST implement MPIC in monitoring mode*"):
- **MUST implement MPIC?** `Yes`
- **Required quorum?:** `Minimally, 2 remote perspectives must be used. If using less than 6 remote perspectives, 1 non-corroboration is allowed. If using 6 or more remote perspectives, 2 non-corroborations are allowed. `
- **MUST block issuance if the quorum is not met:**  `No.`
- **Geographic diversity requirements?:** `Perspectives must be 500km from 1) the primary perspective and 2) all other perspectives used in the quorum. `


\* **Note:** "Monitoring Mode" is a nickname for this period. As opposed to "blocking mode" (described in the next milestone), CAs may still issue a certificate if quorum requirements are not met during this period. This period was intended to offer flexibility to CA owners as they stood up this new capability (i.e., allows for fine tuning implementations). 

------


#### Phase 3: June 15, 2025 - December 14, 2025 ("CAs MUST implement MPIC in blocking mode*"):

- **MUST implement MPIC?** Yes
- **Required quorum?:** Minimally, 2 remote perspectives must be used. If using less than 6 remote perspectives, 1 non-corroboration is allowed. If using 6 or more remote perspectives, 2 non-corroborations are allowed. 
- **MUST block issuance if quorum is not met:**  `Yes.`
- **Geographic diversity requirements?:** Perspectives must be 500km from 1) the primary perspective and 2) all other perspectives used in the quorum. 


\* **Note**: "Blocking Mode" is a nickname. As opposed to "monitoring mode" (described in the last milestone), CAs MUST NOT issue a certificate if quorum requirements are not met from this point forward. 

------


#### Phase 4: December 15, 2025 - June 14, 2026 (requirements strengthen):
- **MUST implement MPIC?** Yes
- **Required quorum?:** Minimally, `3` remote perspectives must be used. If using less than 6 remote perspectives, 1 non-corroboration is allowed. If using 6 or more remote perspectives, 2 non-corroborations are allowed.  
- **MUST block issuance if the quorum is not met:** Yes.
- **Geographic diversity requirements?:** Perspectives must be 500km from 1) the primary perspective and 2) all other perspectives used in the quorum. `Additionally, corroborating perspectives must fall within the service regions of at least 2 distinct regional Internet registries.`

------


#### Phase 5: June 15, 2026 - December 14, 2026 (requirements strengthen):

- **MUST implement MPIC?** Yes
- **Required quorum?:** Minimally, `4` remote perspectives must be used. If using less than 6 remote perspectives, 1 non-corroboration is allowed.   If using 6 or more remote perspectives, 2 non-corroborations are allowed. 
- **MUST block issuance if the quorum is not met:** Yes.
- **Geographic diversity requirements?:** Perspectives must be 500km from 1) the primary perspective and 2) all other perspectives used in the quorum. Additionally, corroborating perspectives must fall within the service regions of at least 2 distinct regional Internet registries.

------


#### Phase 6: December 15, 2026 (requirements strengthen):
- **MUST implement MPIC?** Yes
- **Required quorum?:** Minimally, `5` remote perspectives must be used. If using less than 6 remote perspectives, 1 non-corroboration is allowed. If using 6 or more remote perspectives, 2 non-corroborations are allowed. 
- **MUST block issuance if the quorum is not met:** Yes.
- **Geographic diversity requirements?:** Perspectives must be 500km from 1) the primary perspective and 2) all other perspectives used in the quorum. Additionally, corroborating perspectives must fall within the service regions of at least 2 distinct regional Internet registries.

Reply:
at least for multi-RIR requirement, because remote perspectives are corroborating with primary by definition: 
non-replying RIR was more a question about if dead perspective counted in X remote perspectives used.

Reply:
Looking over the current language I actually feel the RIR requirement is pretty well clarified as being interpreted this way:
https://github.com/ryancdickson/staging/blob/0144a4c0e544173d22a0f09fd4c45d4a52519992/docs/BR.md?plain=1#L1098C29-L1098C29

`
*Effective December 15, 2025*, the CA MUST implement Multi-Perspective Issuance Corroboration using at least three (3) remote Network Perspectives. The CA MUST NOT proceed with certificate issuance if the number of non-corroborations is greater than allowed in the Quorum Requirements table and if the remote Network Perspectives that do corroborate the determinations made by the Primary Network perspective do not fall within the service regions of at least two (2) distinct regional Internet registries.  
`

This seems to clearly state that the 2 RIR service region requirement is 1) only applied to remote Network Perspectives and 2) only applied to perspectives that do corroborate (i.e., non-corroborating perspectives do not count towards diversity).

With this in mind I feel that a dead perspective would count towards X remote perspectives used as long as the CA attempted to reach it, but would not count towards the corroboration count or the 2 distinct RIR requirement. Thus, whether the CA had to initiate a subsequent MPIC request and redial another perspective depends on whether the quorum and the diversity requirement can be satisfied without that perspective.

Is there a particular sentence you would recommend adding to this?

Reply:
no worries. This part of the document recently changed and in past versions I feel this behavior was not as clearly specified